test/e2e/benchmark/throughput.go

```diff
@@ -139,6 +140,17 @@ func TwoNodeBigBlock8MB(logger *log.Logger) error {
 	return runBenchmarkTest(logger, "TwoNodeBigBlock8MB", manifest)
 }
 
+func TwoNodeBigBlock8MBV3(logger *log.Logger) error {
+	manifest := bigBlockManifest
+	manifest.MaxBlockBytes = 8 * testnet.MB
+	// HACKHACK: use a version of celestia-app built from a commit on this PR.
+	// This can be removed after the PR is merged to main.
+	manifest.CelestiaAppVersion = "pr-3882"
+	manifest.TxClientVersion = "pr-3882"
+	manifest.GenesisAppVersion = v3.Version
+	return runBenchmarkTest(logger, "TwoNodeBigBlock8MBV3", manifest)
+}
```

Comment:
Are we adding these benchmarks to see if there's a performance regression? Don't the existing benchmarks use the latest version?

Just trying to understand the point of this.

Reply:
agree, I like the idea of testing this and checking for basic metrics, and I'm guessing the purpose was to sanity check the timeout change with full blocks manually and then remove later so I'm removing these in favor of #3965